tools/install_builder_prerequisites.sh

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env bash
+
+# This file is adapted from Omicron's install_builder_prerequisites.sh.
```

Comment:
This installs libclang as detailed at https://github.com/oxidecomputer/crucible/pull/1778#discussion_r2406836642.